examples/configs/exploration/additive_gaussian.yaml

```diff
@@ -0,0 +1,10 @@
+network: linear
+wrapper:
+  _partial_: True
+  _target_: torchrl.modules.tensordict_module.exploration.AdditiveGaussianWrapper
+  sigma_init: 1.0
+  sigma_end: 0.1
+  annealing_num_steps: 1000
```

Comment:
that's a low number of steps
we should default that to the number of frames, you can do `${collector.total_frames}`